test/quantization/test_quantize.py

```diff
@@ -677,51 +678,71 @@ def checkQuantized(model):
         model = quantize_dynamic(NestedModel().eval(), qconfig_dict)
         checkQuantized(model)
 
+    @override_qengines
     def test_quantized_rnn(self):
         r"""Test dynamic quantization, scriptability and serialization for dynamic quantized lstm modules on int8 and fp16
         """
-        d_in, d_hid = 2, 2
-        model = LSTMDynamicModel().eval()
-        niter = 10
-        x = torch.tensor([[100, -155],
-                          [-155, 100],
-                          [100, -155]], dtype=torch.float).unsqueeze(0).repeat(niter, 1, 1)
-
-        for qengine in supported_qengines:
-            with override_quantized_engine(qengine):
-                for dtype in [torch.qint8, torch.float16]:
-                    if dtype == torch.float16 and qengine == "qnnpack":
-                        # fp16 dynamic quant is not supported for qnnpack
-                        continue
-                    model_quantized = quantize_dynamic(model=model, dtype=dtype)
-
-                    # Smoke test extra reprs
-                    self.assertTrue('DynamicQuantizedLSTM' in str(model_quantized))
-                    self.checkDynamicQuantizedModule(model_quantized.lstm, torch.nn.quantized.dynamic.LSTM, dtype)
-                    self.checkScriptable(model_quantized, [(x, x)], check_save_load=True)
-
-                    class ScriptWrapperPacked(torch.nn.Module):
-                        def __init__(self, cell):
-                            super(ScriptWrapperPacked, self).__init__()
-                            self.cell = cell
-
-                        def forward(self,
-                                    x  # type: PackedSequence
-                                    ):
-                            # type: (...) -> Tuple[PackedSequence, Tuple[torch.Tensor, torch.Tensor]]
-                            return self.cell(x)
-
-                    packed_input = torch.nn.utils.rnn.pack_padded_sequence(x, torch.tensor([10, 5, 2]))
-                    model_with_packed_input = ScriptWrapperPacked(model_quantized.lstm)
-                    scripted = torch.jit.script(model_with_packed_input)
-                    # We cannot trace with input dtype being a packed sequence
-                    self._checkScriptable(model_with_packed_input, scripted, [(packed_input, x)], True)
+        for module_type in ['LSTM']:
+            model = RNNDynamicModel(module_type).eval()
+            niter = 10
+            x = torch.tensor([[100, -155],
+                             [-155, 100],
+                             [100, -155]], dtype=torch.float).unsqueeze(0).repeat(niter, 1, 1)
+
+
+            for dtype in [torch.qint8, torch.float16]:
+                if torch.backends.quantized.engine == 'qnnpack' and dtype == torch.float16:
+                    continue                            # fp16 dynamic quant is not supported for qnnpack
+
+                print(dtype)
```

Comment:
remove print statement?

Reply:
Sure, will clean this up prior to landing